lib/rucio/common/policy.py

```diff
@@ -88,7 +184,8 @@ def get_lifetime_policy():
 
 
 def policy_filter(function):
-    mapping = {'atlas': ['get_scratch_policy', 'archive_localgroupdisk_datasets']}
+    # Ideally the function mapping sdhould not be static, but would be defined in a policy package
+    mapping = {'atlas': ['get_scratch_policy', 'archive_localgroupdisk_datasets', 'get_dest_path']}
```

Comment:
This policy filter is killing the functionality right now for other communities, such as Belle2 with their naming convention. We need to keep this compatible at the moment and also have some kind of generic naming convention in there.
For 1.20 we should bring this into a loadable policy module or so. But right now I think we are removing currently available functionality for them.